docs/content/direct/get-started.md

```diff
@@ -1,38 +1,39 @@
 # Getting Started with KubeStellar
 
-There are multiple ways to get started with Kubestellar, reflecting its flexible architecture that supports various deployment patterns. This page presents two approaches:
+This page shows one concrete example of steps 2--7 from the [full Installation and Usage outline](user-guide-intro.md#the-full-story). This example produces a simple single-host system that is suitable for kicking the tires, using [kind](https://kind.sigs.k8s.io/) to create three new clusters to serve as your KubeFlex hosting cluster and two WECs. This page concludes with forwarding you to one example of the remaining steps. For general setup information, see [the full story](user-guide-intro.md#the-full-story).
```

Comment:
Why is it a KubeFlex hosting cluster?  That's confusing. This is a KubeStellar Core and that language is consistent with (or should be) with the rest of our documentation

Reply:
This is not new text, it has just been moved around a bit.

This text deliberately speaks the same language as the cited "full story". The full story gets into the internal structure of what we call "the core", because the full story is that the consumer of KubeStellar has choices for what they put in their core.